spec/useLoadClientScript.spec.tsx

```diff
@@ -34,6 +34,7 @@ describe("useLoadClientScript", () => {
     await new Promise(nostojs)
 
     hook.rerender()
+    await wait(1)
```

Comment:
Were the assertations expected too fast after re-rerender?

Reply:
yes